lib/openshift/dns.rb

```diff
@@ -0,0 +1,13 @@
+module BushSlicer
+  class Dns < ClusterResource
+	  RESOURCE = "dns"
```

Comment:
Please fix the indention, there are multiple inconsistent intentions in this PR.

Reply:
Done